frontend/app/src/store/main.ts

```diff
@@ -1945,6 +1972,57 @@ export class MainStore {
       return [];
     }
   }
+
+  async getInvoiceDetails(payment_request: string): Promise<InvoiceDetails | InvoiceError> {
+    try {
+      const r: any = await fetch(`${TribesURL}/gobounties/invoice/${payment_request}`, {
+        method: 'GET',
+        mode: 'cors',
+        headers: {
+          'Content-Type': 'application/json'
+        }
+      });
+      return r.json();
+    } catch (e) {
+      console.log('Error gettHistories', e);

```

Comment:
console.warn

Reply:
well I think that message is incorrect too it should be `Error getInvoiceDetails`